src/Host/Protocol/Impl/RUserProfile/RUserProfileServices.cs

```diff
@@ -12,16 +12,22 @@
 using Microsoft.Common.Core.Json;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
+using System;
 
 namespace Microsoft.R.Host.Protocol {
-    public class RUserProfileCreator {
+    public class RUserProfileServices {
```

Comment:
This class, as well as the one below, should not be in the `Protocol` assembly. `Protocol` is only for interfaces and DTOs that are shared between the broker, the client, and the user profile service, and the associated infrastructure like JSON converters.  These two belong to `R.Host.UserProfile`.